cloudant-client/src/main/java/com/cloudant/client/internal/views/RowImpl.java

```diff
@@ -58,11 +58,19 @@ public V getValue() {
 
     @Override
     public Document getDocument() {
-        return gson.fromJson(row.get("doc"), Document.class);
+        Document doc = null;
```

Comment:
I just noticed that it would be a lot easier if this just called `return getDocumentAsType(Document.class);`
then the `row` and `error` checks need only happen in one place.

Reply:
Fixed in 21bfc3c.